doc/sphinx/language/core/index.rst

```diff
@@ -30,10 +30,67 @@ will have to check their output.
 .. toctree::
    :maxdepth: 1
 
+   .. The two chapters Gallina specification language and CIC are
+      planned for removal.  Eventually, the present "Core language"
+      chapter will play the role of the former Gallina chapter, except
+      that some core notions will have moved there from the former
+      Gallina extensions chapter.
+
    ../gallina-specification-language
    ../cic
+
+   .. Comments are used to show upcoming sections (not yet split out).
+
+   .. Lexical conventions, how to read the grammars in the manual,
+      presentations of the basic notions: terms, commands, tactics,
+      attributes and options.
+
+   .. Terms, typing and conversion (only basic constructs:
+      abstraction, product, application, type cast, brief mention of
+      sorts).
+
+   .. Assumptions
+
+   .. Definitions (vernac) and local definitions (let in).  Includes
+      opaque definitions and a brief introduction to the interactive
+      proof mode, in the spirit of CEP #42.
```

Comment:
Wouldn't interactive proof mode naturally go in the Writing Proofs chapter?  IIUC it's not a core language feature.

Reply:
Yes, I just meant briefly write that it exists and that link to the chapter where it is docummented.